docs/news.html

```diff
@@ -7,15 +7,15 @@
   </head>
   <body>
     <div class="news-item">
-      <span class="item-date">2024/12/17</span>
-      <a href="https://forum.getodk.org/t/odk-central-v2024-3-year-end-goodies/51515" target="_blank">
-        ODK Central v2024.3
+      <span class="item-date">2025/1/13</span>
```

Comment:
I'd say 01 instead of 1 just to match what we do in Frontend:

```suggestion
      <span class="item-date">2025/01/13</span>
```